es-app/src/FileFilterIndex.cpp

```diff
@@ -171,6 +171,9 @@ std::string FileFilterIndex::getIndexableKey(FileData* game, FilterIndexType typ
 			key = Utils::String::toUpper(game->metadata.get("kidgame"));
 			break;
 		}
+		default:
```

Comment:
Added default case to resolve compiler warning:

/home/devel/temp/EmulationStation_Stable/es-app/src/FileFilterIndex.cpp:91:9: warning: enumeration va
lue 'NONE' not handled in switch [-Wswitch]
        switch(type)
               ^~~~
1 warning generated.